src/frontend/Felles/ScrollToTop/ScrollToTop.tsx

```diff
@@ -5,7 +5,16 @@ export default function ScrollToTop() {
     const { pathname } = useLocation();
 
     useEffect(() => {
-        window.scrollTo(0, 0);
+        const scrollTilTopp = () => {
+            const container = document.querySelector('#scroll-topp');
+            if (container) {
+                container.scrollTop = 0;
+            } else {
+                window.scrollTo(0, 0);
```

Comment:
Denne scroller hovedvinduet til toppen, men ikke alle underliggende containere?

Reply:
![scroll-området](https://github.com/user-attachments/assets/be1de6eb-84a0-428f-a27e-6babb61e04e4)
Det er containeren i rødt som vil gå til topp

Reply:
Skjønte det, men hva er vitsen med else-branchen?

Reply:
Vi ha en del sider som IKKE er fokusert rundt én behandling - dokumentoversikt, vedtaksperioder, behandlingsoversikt osv. For alle disse sidene vil `window.scrollToTop(...)` fungere når du går mellom sider. Og man ønsker jo å scrolle til topps, og ikke bli på en tilfeldig scrollsted når man bytter sider i løsningen :)